scripts/data-migrations/purger-sessions-milo-early-adopters.js

```diff
@@ -0,0 +1,36 @@
+require('dotenv').config()
+const { Sequelize } = require('sequelize')
+
+const sequelize = new Sequelize(process.env.DATABASE_URL, {
+  dialect: 'postgres'
+})
+
+const ID_STRUCTURE_EARLY_ADOPTERS_1 = '80620S00'
+
+sequelize.transaction(async transaction => {
+  const jeunesEarlyAdopters = (
+    await sequelize.query(
+      `SELECT * FROM jeune WHERE id_structure_milo IN (?)`,
+      {
+        replacements: [ID_STRUCTURE_EARLY_ADOPTERS_1],
+        transaction
+      }
+    )
+  )[0]
+
+  for (jeune of jeunesEarlyAdopters) {
```

Comment:
Pourquoi un `for ... of` et pas une jointure dans la requête de suppression ?

Reply:
flemme